go.mod

```diff
@@ -1,10 +1,10 @@
 module github.com/jwhited/wgsd
 
-go 1.14
+go 1.15
```

Comment:
what's the reasoning for changing the expected version? I don't think it's the wgctrl bump since wgctrl specifies 1.13

https://golang.org/ref/mod#go-mod-file-go